storage/flux/reader.go

```diff
@@ -395,54 +395,32 @@ func convertGroupMode(m query.GroupMode) datatypes.ReadGroupRequest_Group {
 }
 
 const (
-	startColIdx = 0
-	stopColIdx  = 1
-	timeColIdx  = 2
-	valueColIdx = 3
+	startColIdx         = 0
+	stopColIdx          = 1
+	timeColIdx          = 2
+	windowedValueColIdx = 2
+	valueColIdx         = 3
```

Comment:
I'm not seeing where the `valueColIdx` get used now. Is there a purpose for it or is it just being left for future changes?

Reply:
It is actually used a lot. See `determineTableColsForGroup()` and `determineTableColsForSeries()`